### ISSUE TITLE
Option to open onehanded mode in lefthanded position

### ISSUE THREAD

**Issue by delthia** [florisboard/florisboard#2036]
Hi,
As it is possible to, after putting the keyboard in one-handed mode, change it to be comfortable with the left hand, I think it would be helpful to be able to directly open it in this way, either because you set it in the settings, or because it remembers the last position you used the one-handed mode in.

I think that this would improve usability for lefthanded people, like it has already been done with other features, for example being able to flip the smartbar buttons.

Thanks

**Comment by Glitchy-Tozier** [comment]
An elegant way to accomplish something like this without introducing more complexity to the settings might be to _make the keyboard remember_ which side the one-handed mode was used last.

**Comment by delthia** [comment]
Correct, that was what I was thinking would be the easiest in terms of usability.
Now, I can only propose it, as kotlin is not within the languages I know.

**Comment by patrickgold** [comment]
> An elegant way to accomplish something like this without introducing more complexity to the settings might be to make the keyboard remember which side the one-handed mode was used last.

That sounds like the best approach to this problem.
